index.js

```diff
@@ -16,13 +20,21 @@ module.exports = {
 
     if (!process.env.EMBER_CLI_FASTBOOT) {
       if (app.env === "production") {
-        app.import(app.bowerDirectory + '/hammer.js/hammer.min.js');
+        app.import('vendor/hammer.min.js');
       } else {
-        app.import(app.bowerDirectory + '/hammer.js/hammer.js');
+        app.import('vendor/hammer.js');
       }
     }
   },
 
+  treeForVendor(vendorTree) {
+    let hammerTree = new Funnel(path.dirname(require.resolve('hammerjs/hammer.js')), {
```

Comment:
u can use require.resolve('hammerjs').main in the future (if main prop in package.json points to that same file)